cg/models/instrument_run_directory_data/instrument_run_directory_data.py

```diff
@@ -36,7 +36,11 @@
 }
 
 
-class FlowCellDirectoryData:
+class InstrumentRunDirectoryData:
+    pass
+
+
+class IlluminaRunDirectoryData(InstrumentRunDirectoryData):
```

Comment:
As agreed on the standup
```suggestion
class IlluminaRunDirectory(InstrumentRunDirectoryData):
```

Reply:
I see it was changed, resolving